packages/devtools_app/lib/src/shared/diagnostics/inspector_service.dart

```diff
@@ -7,6 +7,9 @@
 // If you add methods to this class you should also add them to
 // InspectorService.java.
 
+/// @docImport '../../screens/performance/panes/rebuild_stats/rebuild_stats_model.dart';
```

Comment:
hmm this seems suspicious. Why are we referencing code for the Performance page as part of the Inspector service?

Reply:
Yeah, sorry. It's very awkward that these fix CLs don't indicate what comment reference is newly resolved with a doc-import.

In this case, it's the comment for `whichLocationIdMap`:

> /// See [LocationMap] which provides support to parse this JSON.